app/db/models.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+
+const marketSchema = new mongoose.Schema({
+  _id: String,
+  personalInfo: {
```

Comment:
`personalInfo` is a bit misleading, rename to something that mentions it's the evaluator's personal info e.g. `lastAssessedBy` or something